launcher/cli/setup.go

```diff
@@ -41,7 +42,7 @@ func setup() {
 	}
 
 	go func() {
-		listenAddr := "localhost:6060"
+		listenAddr := viper.GetString("global-pprof-listen-addr")
```

Comment:
The flag usage specifies `if non-empty`, I don't think we handle this case correctly here.